docs-site/content/docs/help/faq.md

```diff
@@ -16,7 +16,7 @@ flair =[]
 
 <details>
 <summary>How can I automatically reload code?</summary>
-Try [cargo watch](https://crates.io/crates/cargo-watch):
+Try (cargo watch)[https://crates.io/crates/cargo-watch]:
```

Comment:
looks like this should be the opposite brackets?

Reply:
yes, you are right. I've changed it because it was not rendering, but issue was different: [the rule of adding newlines before and after blocks](https://docs.gitlab.com/ee/user/markdown.html#details-and-summary:~:text=Remember%20to%20leave%20a%20blank%20line%20before%20and%20after%20any%20Markdown%20sections) was not respected. I will fix it with next commit.